lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -259,8 +259,10 @@ public function get_items( $request ) {
 		$request_params = $request->get_query_params();
 		if ( ! empty( $request_params['filter'] ) ) {
 			// Normalize the pagination params.
-			unset( $request_params['filter']['posts_per_page'] );
-			unset( $request_params['filter']['paged'] );
+			unset(
+				$request_params['filter']['posts_per_page'],
+				$request_params['filter']['paged']
+			);
```

Comment:
I have changed the PR to keep it in one line (with only one `unset()`).

The line is long, yes, but it is actually shorter than the one directly below... :)